From 3dd4933aa02ea87a0e3bf3923b7655c035d6ba33 Mon Sep 17 00:00:00 2001
From: Jeffery <jeffery_wu@tw.shuttle.com>
Date: Thu, 24 Nov 2016 16:16:47 +0800
Subject: [PATCH 371/443] If Device Name is not empty string while "Creating",
 it should be not duplicated. Note: rollback Frank's changed code on 11/22

---
 devices/views.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/devices/views.py b/devices/views.py
index 4bc4a6d..09f0027 100755
--- a/devices/views.py
+++ b/devices/views.py
@@ -132,6 +132,8 @@ def add_device(request):
         Name = request.POST.get('Name', '')
         MAC = request.POST.get('MAC', '')
         IP = request.POST.get('IP', '')
+        if Name != '' and Devices.objects.filter(Name = Name).exists():
+            return JsonResponse(res_code[2])
         if MAC != '' and Devices.objects.filter(MAC = MAC).exists():
             return JsonResponse(res_code[3])
         if IP == '' or Devices.objects.filter(IP = IP).exists():
-- 
1.9.1

